prow/config/jobs/tools.yaml

```diff
@@ -30,6 +30,12 @@ jobs:
     regex: 'docker/.+|cmd/.+'
     requirements: [gcp, docker]
 
+  - name: perf-test-validation
```

Comment:
we should only run this on relevant changes. otherwise it will run for every change. You can do this with `regex` like above

Reply:
yeah, thanks for pointing this out, let me figure out how can we identify something in the istio/tools PR as a relevant change.